code/lib/docs-tools/package.json

```diff
@@ -56,6 +56,7 @@
   "publishConfig": {
     "access": "public"
   },
+build/lib-docs-tools+lib-instrumenter-tsup
```

Comment:
Don't remember adding this line in, but I know it shouldn't be there. Also not sure if that made it fail the tests on my end.  Let me know if there's anything I can do